src/sssom_schema/schema/sssom_schema.yaml

```diff
@@ -85,6 +85,9 @@ types:
     uri: rdfs:Resource
     see_also:
       - https://mapping-commons.github.io/sssom/spec/#tsv
+ AbsoluteURI:
+    typeof: uri
+    description: An absolute URI as per RFC 3986.
```

Comment:
Could you please give a set of examples and counterexamples to make it clear what sort of things are being avoided here? 

Further, I think it would be nice to give the reader a quick summary of what RFC 3986 entails